src/Abc.Zebus.Tests/Abc.Zebus.Tests.csproj

```diff
@@ -3,6 +3,8 @@
     <TargetFrameworks>net471;netcoreapp2.1</TargetFrameworks>
     <Version>$(ZebusVersion)</Version>
     <IsTestProject>true</IsTestProject>
+    <OutputType>Exe</OutputType>
+    <StartupObject>Abc.Zebus.Tests.Program</StartupObject>
```

Comment:
That will prevent tests from running on .NET Core, as the runner defines its own entry point.